KDIC/FeedViewController.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+class ScheduleViewController: UIViewController ,  UITableViewDataSource {
+    @IBOutlet weak var daySegmntCntrl: UISegmentedControl!
+    
+    var dayScheduleKeysArray = [AnyObject]()
+    var dayScheduleValuesArray = [AnyObject]()
+    
+    @IBOutlet weak var scheduleTableView: UITableView!
+    
+    var kdicScrapperReqs: KDICScrapperReqs?
+    
+    @IBOutlet weak var scheduleDataLoadingIndicator: UIActivityIndicatorView!
+    
+    var scheduleUrlStr = "http://kdic.grinnell.edu/scheduleScript.php"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.kdicScrapperReqs = KDICScrapperReqs(urlStr: scheduleUrlStr, tableView: scheduleTableView, completionCall: loadInitialTableData)
+        self.kdicScrapperReqs?.get_data_from_url()
+
+    }
+    
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return dayScheduleKeysArray.count
+    }
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithIdentifier("ScheduleCell", forIndexPath: indexPath) as! ScheduleTableCell
+        
+        let showTime = self.dayScheduleKeysArray[indexPath.row] as! String
+        let showName = self.dayScheduleValuesArray[indexPath.row] as! String
```

Comment:
Make a `Show` class. It will simplify everything the same way the `Article` class did.